src/mono/mono/mini/interp/interp.c

```diff
@@ -2145,6 +2145,12 @@ typedef struct {
 	gpointer *many_args;
 } InterpEntryData;
 
+static gboolean
+mono_interp_is_method_multicastdelegate_invoke (MonoMethod *method)
```

Comment:
I would drop the `mono` prefix for static methods. Maybe even the `interp` prefix for such small methods that are not really interp related.

Reply:
it becomes non-static for the jiterpreter once it starts being used by it